srv/isc-agent/plugins/tcp/http/main.py

```diff
@@ -13,6 +18,15 @@ def render_GET(self, request: Request):
         content = f"I am request #{self.numberRequests}\n"
         return content.encode("ascii")
 
+    def render_HEAD(self, request: Request):
+        request.setResponseCode(HTTPStatus.OK)
+        request.setHeader(b"Server", f"{PRODSTRING}")
```

Comment:
If the PRODSTRING is the only value of the string it doesnt need to be in quotes. You can pass the string directly.